spec/models/user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { described_class.new(name: 'John') }
+
+  describe 'associations' do
+    it { should have_many(:posts).with_foreign_key(:author_id).dependent(:destroy) }
```

Comment:
- **[OPTIONAL]** This test returns an error, please fix it to pass them all.

![image](https://github.com/Zeph180/blog-it/assets/58642949/8a5b6a61-f4a0-4c42-b54d-53e91a52c09a)